modules/data-loading/pages/data-streaming-connector/big-query.adoc

```diff
@@ -0,0 +1,269 @@
+You can also supply other optional properties to the connection that are supported by https://storage.googleapis.com/simba-bq-release/jdbc/Simba%20Google%20BigQuery%20JDBC%20Connector%20Install%20and%20Configuration%20Guide_1.3.0.1001.pdf[the BigQuery JDBC connector]:
+
+[source.wrap,text]
+----
+connection.url=jdbc:bigquery://<host_url>:<port_number>;<property1>=<value1>;<property2>=<value2>;...
+----
+
+
+|===
+|Parameter |Description
+
+|`host_url`
+|Domain name or IP address of the BigQuery server.
+Required.
+
+|`port_number`
+|Number of the TCP port to connect to.
+This is usually 443.
+Required.
+
+|`project_id`
+|ID of your Google Cloud Project for you BigQuery server.
+Required.
+
+|`AuthValue`
+a|Required. A number that specifies the type of authentication used by the connector:
+
+* `0`: Authentication with Google Service Account.
+This requires that you supply two additional parameters:
+** `OAuthServiceAcctEmail`
+** `OAuthPvtKeyPath`
+* `1`: Authentication with access token or refresh token.
+This requires that you supply one of the two following parameters:
+** `OAuthAccessToken`
+** `OAuthRefreshToken`
+
+|`OAuthServiceAcctEmail`
+|Email Google Service Account used to authenticate the connection.
+Required if you set `AuthValue` to `0`.
+
+|`OAuthPvtKeyPath`
+|Path to the key file that is used to authenticate the service account email address.
+This parameter supports keys in .p12 or .json format.
+Required if you set `AuthValue` to `0`.
+|`OAuthAccessToken`
+|Access token of your Google Service Account.
+See Google Cloud documentation to https://cloud.google.com/iam/docs/create-short-lived-credentials-direct#sa-credentials-oauth[generate an access token].
+Required if you set `AuthValue` to `1`.
+|`OAuthRefreshToken`
+|Refresh token of your Google Service Account.
+See Google Cloud documentation to https://developers.google.com/identity/protocols/oauth2/web-server#exchange-authorization-code[generate a refresh token].
+Required if you set `AuthValue` to `1`.
+|===
+
+==== Value converter
+BigQuery stores data in a proprietary columnar format.
+In order for TigerGraph to ingest the data, Data Streaming Connector uses a value converter to convert the data into the CSV format.
+To do this, add a line in the configuration file that specifies the value converter:
+
+[source.wrap,text]
+value.converter=com.tigergraph.kafka.connect.converters.TigerGraphCsvConverter
+
+==== Subsection configurations
+Subsection configurations are specified after the overall connector configurations.
+Each set of subsection configurations creates an instance of a connector by specifying a connector name and a Kafka topic that the connector streams messages to.
+
+When streaming from BigQuery, each subsection also must have a SQL query statement used to query your BigQuery warehouse.
+The results of the query are streamed to the corresponding topic.
+
+NOTE: If you are querying `STRUCT` data or arrays, see
```

Comment:
Update: besides the `STRUCT` and `ARRAY` type, another one we need to mention:
since TG doesn't support type `bytes`, if querying bigquery `BYTES` type, it will be converted to UTF8 string.